README.md

```diff
@@ -311,25 +311,27 @@ const ThemeChanger = () => {
 To fix this, make sure you only render UI that uses the current theme when the page is mounted on the client:
```

Comment:
For this change to make sense we need to update the code block above it too, so it's easier to see the difference between hydration safe/unsafe code (the `mounted` state)